Sources/Public/CalendarView.swift

```diff
@@ -1255,6 +1255,8 @@ private final class ScrollViewDelegate: NSObject, UIScrollViewDelegate {
   // MARK: Internal
 
   func scrollViewDidScroll(_ scrollView: UIScrollView) {
+    guard let calendarView else { return }
+
     calendarView.preventLargeOverScrollIfNeeded()
```

Comment:
is is the line it was crashing on. It's somewhat related to the new intrinsic content size calculation I added last week https://github.com/airbnb/HorizonCalendar/pull/250/files#diff-733b4844a228b4fda6b27e4f88b42a6fc4b04eb589081da658741416a2152bd9R1079-R1101

But I'm not sure what the issue is here. Everything works fine in a real environment vs. test.